docs/API/V1/README.md

```diff
@@ -0,0 +1,86 @@
+
+## Discovering the Pbench server API
+
+Once you know the hostname of a Pbench server, you can ask for the API configuration using the [endpoints](endpoints.md) API. This will report the server's version and a list of all API end points (URI roots) supported by the server.
+
+## Login and registration
+
+You can register a new user (depending on the administration policy of the server) using the [register](register.md) API. If this succeeds, you can log in
+
+You can log in as a known username by calling the [login](login.md) API, which returns a bearer schema authentication token supplied to subsequent API calls using the `authorization` header.
+
+You can log out an active authentication token by passing it as the `authorization` header to the [logout](logout.md) API.
+
+While logged in, you can retrieve (`GET`) and modify (`PUT`) your user profile through the [user](user.md) API.
+
+## Dataset metadata
+
+You can read a more complete specification of Pbench server metadata at [metadata](../metadata.md).
+
+When a dataset is first processed, the Pbench server will populate basic metadata, including the creation timestamp, the owner of the dataset (the user authorization token used to authenticate the `PUT` from the agent via `pbench-results-move`), and the full contents of the dataset's `metadata.log` file inside the dataset tarball. The Pbench server will also calculate a default deletion date for the dataset based on the owner's retention policy and the server administrator's retention policy.
+
+Clients can also set arbitrary metadata through the "dashboard" and "user" metadata namespaces. The "dashboard" namespace can only be modified by the owner of the dataset, and is visible to anyone with read access to the dataset. The "user" namespace is private to each authenticated user, and even if you don't own a dataset you can set your own private "user" metadata to help you categorize that dataset and to find it again.
+
+The primary dataset `resource_id` is the computed MD5 of the dataset tarball. This is generated by the agent, and checked on the server side to ensure data consistency.
+
+## Discovering accessible datasets
+
+The datasets accessible by a client are limited by the dataset access controls and the client's authorization. Any client can read all "public" datasets, but only the owning user can access "private" datasets.
+
+You can determine the datasets you're allowed to view using the [list](list.md) API. This will return the resource name, the formal resource ID, and selected metadata for each dataset the authenticated user is allowed to read. You can filter the datasets you want listed by date range, owning user, or access policy.
+
+It may sometimes be convenient to know the date range of those datasets, without retrieving the entire list and processing the `dataset.created` timestamp for each. You can do this by calling the [daterange](daterange.md) API; for example, to establish a date picker control.
```

Comment:
I guess I kind of keyed on the "It may sometimes be convenient to know the YYY, without retrieving the entire XXX", and that sounded very much like the description of a `HEAD` operation, to me.  If you really want it to be a `GET` without a body, then that's fine -- just add the pertinent headers to _both_ responses.  🙃

Or, we can just provide the extra API, but it seemed like there might be a savings here.